CMakeLists.txt

```diff
@@ -141,7 +141,7 @@ message ("linker Flags: " ${CMAKE_EXE_LINKER_FLAGS})
 # Configuration
 ########################################
 
-set(CMAKE_C_STANDARD 99)
+set(CMAKE_C_STANDARD 11)
```

Comment:
We need to keep C99 standard

Reply:
The codebase already uses C11 features, so either the compiler needs to be set to use C11 or the code using C11 features has to be changed. Are there any plans to do the latter or do you want an external PR to address this?

Reply:
#124

Reply:
I split this part out into #125.